packages/cli/src/outputting.js

```diff
@@ -38,9 +38,14 @@ export function globalOperation(msg) {
 /**
  * @param {string} msg
  * @param {import('@bilt/types').PackageInfo} packageInfo
+ * @param {number | null} current
+ * @param {number | null} length
  */
-export function packageHeader(msg, packageInfo) {
-  outputFunction(chalk.greenBright.underline(`**** [${packageInfo.directory}] ${msg}`))
+export function packageHeader(msg, packageInfo, current = null, length = null) {
```

Comment:
Why would a `null/undefined` be passed here? All calls to `packageHeader` will probably pass both `current` and `length`.